scripts/run_e2e_tests.py

```diff
@@ -422,21 +369,10 @@ def main(args=None):
                     'Not rerunning as per instructions from '
                     'logging server.')
                 break
-            # No rerun override, so follow rerun policies.
-            elif policy == RERUN_POLICY_NEVER:
+            else:
                 print(
-                    'Not rerunning because the policy is to never '
-                    'rerun the {} suite'.format(parsed_args.suite))
+                    'Not rerunning since no instructions from logging server.')
```

Comment:
Is this an unexpected case (would we actually expect the logging server to respond)?

If it's unexpected, I think maybe rephrasing as "Not rerunning since the logging server is not responding." might be slightly better framing. Do we timeout in that case?

Reply:
At this point we know the logging server responded, but it responded with `None` to indicate "no instruction" instead of `True` (yes, you should rerun) or `False` (no, you should not rerun). When the logging server responds `None`, it means that `run_e2e_tests.py` should decide whether to rerun (with the current code, it will always default to not rerunning).

The logging server doesn't ever respond with `False` right now, but the next step in moving the rerun logic to the logger will be to add that capability.

Reply:
Hm, this feels like a somewhat odd contract to me. Couldn't the decision on whether to rerun or not always be contained in one place (either here or the logging server) -- why is there a need to split responsibility?

Reply:
done